modules/vba_documents/app/mailers/vba_documents/unsuccessful_report_mailer.rb

```diff
@@ -9,9 +9,17 @@ def self.fetch_recipients
       env = 'prod' if env.eql?(:unknown_environment)
       # the above shouldn't get hit, but if an environment becomes undetectable we will send that env to everyone
       # and tag it as 'unknown_environment' to motivate a quick repair.
-      hash = YAML.load_file("#{__dir__}/unsuccessful_report_recipients.yml")
-      env_hash = hash[env.to_s].nil? ? [] : hash[env.to_s]
-      env_hash + hash['common']
+      recipients = YAML.load_file("#{__dir__}/unsuccessful_report_recipients.yml")
+
+      recipients_for_current_env = recipients[env.to_s] || []
+      recipients_for_all_envs = recipients['common'] || []
+
+      if Settings.vba_documents.slack.enabled
+        slack_alert_email = Settings.vba_documents.slack.default_alert_email
+        (recipients_for_current_env + recipients_for_all_envs).uniq.append(slack_alert_email).compact
+      else
+        (recipients_for_current_env + recipients_for_all_envs).uniq
+      end
```

Comment:
You'll notice the duplicated code here. Next time I touch these files I plan to move the "fetch recipients" functionality to a separate Concern.